c/parallel/src/transform.cu

```diff
@@ -54,44 +54,70 @@ constexpr auto output_iterator_name = "output_iterator_t";
 
 struct transform_runtime_tuning_policy
 {
+  int min_bif;
   int block_threads;
   int items_per_thread_no_input;
   int min_items_per_thread;
   int max_items_per_thread;
+  int items_per_thread_vectorized;
+  int load_store_word_size;
+
+  int MinBif() const
+  {
+    return min_bif;
+  }
 
   // Note: when we extend transform to support UBLKCP, we may no longer
   // be able to keep this constexpr:
   static constexpr cub::detail::transform::Algorithm GetAlgorithm()
   {
+    // FIXME(bgruber): this needs to incorporate the algorithm selection logic from CUB policy_hub
     return cub::detail::transform::Algorithm::prefetch;
   }
```

Comment:
Do we want to take the opportunity and turn this into a static constexpr function?

No change requested, can be done as follow up